privacyidea/api/token.py

```diff
@@ -113,6 +114,8 @@
 @prepolicy(check_token_init, request)
 @prepolicy(init_tokenlabel, request)
 @prepolicy(enroll_pin, request)
+@prepolicy(twostep_enrollment_activation, request)
+@prepolicy(twostep_enrollment_parameters, request)
```

Comment:
The order of the two decorators has to be vice versa. Otherwise

    if is_true(getParam(request.all_data, "2stepinit", optional)):

in enroll_parameters will not work as expected.